cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### Sentences
+
+The constitution text MUST only contain at maximum one sentence per line.
+Each sentence MUST be followed by a newline.
+
+Each sentence SHOULD start on its own line with a capitalized letter.
+
+Example:
+
+```md
+This is a short sentence on one line.
+
+This is a long sentence and I have valid reasons for it being so long,
+such as being an example of a long sentence.
+```
+
+It is recommended that each line is ended by either a full stop or comma.
+
+### File Type
+
+The constitution file MUST be a text file named `constitution.txt`.
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text MUST be in its raw text, including any [Rich Text Formatting](#rich-text-formatting) related characters.
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism such as IPFS.
+Where backups can be easily made in a permissionless manner by interested parties.
+This storage platform MUST be easily accessible, with strong tooling support.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+// only ascii
+
+### Rich Text Formatting
+
+The constitution text MAY include a strict subset of rich text styling as defined in this specification.
+Tooling rendering the constitution SHOULD recognize these and render them faithfully.
+
+#### Headers
+
+Headers are denoted via a hashtag character `#` followed by a space ` `.
+There are six levels to headers with, each being defined via a extra `#`.
+Headers are ended via a line break.
+Headers SHOULD be followed below by a blank line.
+Headers SHOULD not be preceded by whitespace.
+
+The lower the number of `#` the larger order the text SHOULD be rendered.
+
+Example:
+```md
+# H1
+
+## H2
+
+### H3
+
+#### H4
+
+##### H5
+
+###### H6
+
+```
+
+If text is in a header no other formatting can be applied.
+
+#### Emphasis
+
+Emphasis is applied to text between single or double asterisks, without space between asterisks and text.
+Italicized emphasis is shown via single asterisk (`*`).
+Bold emphasis is shown via double asterisks (`**`).
+
+Emphasis cannot span multiple lines.
+
+Examples:
+```md
+Emphasis, aka italics, with single *asterisks*.
+
+Strong emphasis, aka bold, with double **asterisks**.
+```
+
+The text contained within headings can be emphasized.
+
+Both italicized and bold cannot be applied to the same text.
+
+#### Code and Syntax Highlighting
+
+Texted can be highlighted as code, when encased without spaces by backticks.
+This MUST not contain line breaks.
```

Comment:
@Ryun1 can we be sure a constitution will not contain code blocks?

Reply:
not if it follows this CIP 😆

Reply:
I guess I should have said, "How could we be sure there will _not_ be a convincingly proposed constitution that contains code blocks?" 🤔

Reply:
I dont see it being necessary
but also cant predict it

Reply:
maybe... but I wasn't thinking in terms of missing markdown features... rather, possible instances of a constitution in which a code block could indicate some vital feature of governance that inline code could not.

Inline code tags would be generally used for machine readable terms... especially blockchain parameters... but what if a simple procedure needed to be indicated?  Not something technical like Plutus or Javascript, but pseudocode as for a simple flowchart or high level algorithm?  Would all constitutions be required to indicate these procedures with nested lists and such?